src/reducers/userDataReducers.js

```diff
@@ -27,8 +27,7 @@ function userDataReducers(state = initialState, action) {
     case CLEAR_USER_DATA:
       return initialState;
     case LOAD_FAILURE:
-      state.error = "Failed to load user data...";
-      return state;
+      return Object.assign({}, state, { error: "Failed to load user data..." });
```

Comment:
suggestion: have the error message be customizable for future use. Cuse now all errors will say failed to load user data

Reply:
So that means you'll have to update the calls in app.js to pass in a string